swri_transform_util/src/local_xy_util.cpp

```diff
@@ -272,7 +272,11 @@ namespace swri_transform_util
       double rlat = latitude * swri_math_util::_deg_2_rad;
       double rlon = longitude * swri_math_util::_deg_2_rad;
       double dLat = (rlat - reference_latitude_) * rho_lat_;
-      double dLon = (rlon - reference_longitude_) * rho_lon_;
+      double dLon = (rlon - reference_longitude_);
+      // Check for case where the shortest distance is across the date line
```

Comment:
Just a minor point: I think you mean the 180th meridian or the antimeridian (I had to look that up to be sure) rather than the date line, which only partially coincides with the antimeridian: https://en.wikipedia.org/wiki/International_Date_Line.

Reply:
Yes. My use of the term dateline was not accurate.